script.js

```diff
@@ -12,10 +13,21 @@ const searchFilteredEpisodes = (input) => {
     epi.summary.toLowerCase().includes(input.toLowerCase())
```

Comment:
It's better to use the full word for the variable name, in this case episode is better than epi. this makes it clearer and easier to understand what your code is doing.